plugins/InspectionGadgets/src/com/siyeh/ig/redundancy/RedundantStringOperationInspection.java

```diff
@@ -881,6 +960,92 @@ public void applyFix(@NotNull Project project, @NotNull ProblemDescriptor descri
     }
   }
 
+  private static class StripEqualsToIsBlankFix implements LocalQuickFix {
+    private final PsiExpression wholeExpression;
```

Comment:
Quick-fix cannot store PSI elements in the fields. I believe there should be a devkit inspection that warns about this

Reply:
Done